lgc/patch/SystemValues.cpp

```diff
@@ -83,11 +83,9 @@ Value *ShaderSystemValues::getEsGsRingBufDesc() {
     // Ensure we have got the global table pointer first, and insert new code after that.
     BuilderBase builder(getInternalGlobalTablePtr()->getNextNode());
     m_esGsRingBufDesc = loadDescFromDriverTable(tableOffset, builder);
-    if (m_shaderStage != ShaderStageGeometry && m_pipelineState->getTargetInfo().getGfxIpVersion().major >= 8) {
```

Comment:
Are you sure that the `m_shaderStage` check can be removed? Does PAL set this unconditionally now perhaps?

If in doubt, perhaps back out this one change from the PR and make it a separate change.

Reply:
Good catch. It is a wrong removal. `m_shaderStage` cannot be removed.